auth0_flutter/EXAMPLES.md

```
@@ -700,6 +701,51 @@ final credentials = await auth0.credentialsManager.credentials();
+final credentials = await auth0.credentialsManager.credentials();
+
+// null when the connection option is not enabled, or the session predates the
+// feature. A null value means "no upstream ceiling".
+final sessionExpiry = credentials.sessionExpiry;
+if (sessionExpiry != null) {
+  print('Upstream IdP session ends at: $sessionExpiry');
+}
+```
+
+> ℹ️ Enforcement applies a small negative leeway (about 30 seconds) for clock skew, so the session is treated as expired slightly **before** this exact timestamp. Account for this if you build a countdown off `sessionExpiry`, otherwise the session may end a little earlier than the value shown.
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)